.npmignore

```diff
@@ -0,0 +1,2 @@
+.idea
```

Comment:
`.idea` already in `.gitignore`

Reply:
Without this line `.idea` content goes to package

```
npm notice loud-stylelint-config git:(chore/package_content) ✗ npm pack --dry-run 
npm notice 📦  @yandex-cloud/stylelint-config@1.1.1
npm notice === Tarball Contents === 
npm notice 128B  .idea/inspectionProfiles/profiles_settings.xml
npm notice 252B  .idea/inspectionProfiles/Project_Default.xml  
npm notice 4.3kB .idea/misc.xml                                
npm notice 310B  .idea/modules.xml                             
npm notice 180B  .idea/vcs.xml                                 
npm notice 458B  .idea/yandex-cloud-stylelint-config.iml       
npm notice 1.9kB CONTRIBUTING.md                               
npm notice 1.1kB LICENSE                                       
npm notice 686B  README.md                                     
npm notice 4.6kB index.js                                      
npm notice 8.9kB order.js                                      
npm notice 594B  package.json                                  
npm notice 189B  prettier.js                                   
npm notice === Tarball Details === 
npm notice name:          @yandex-cloud/stylelint-config          
npm notice version:       1.1.1                                   
npm notice filename:      @yandex-cloud/stylelint-config-1.1.1.tgz
npm notice package size:  6.1 kB                                  
npm notice unpacked size: 23.6 kB                                 
npm notice shasum:        3c4f45c802987bb09ddbe51dd8371225b02741b9
npm notice integrity:     sha512-MJdQjy2dX0Obk[...]6ghnG3AxIigeQ==
npm notice total files:   13                                      
npm notice 
yandex-cloud-stylelint-config-1.1.1.tgz
```